### PR TITLE
Integrate model with controller of python compiler 

### PR DESCRIPTION
**Pull request by JRojas999**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/controller/PythonCompilerController.java [hunk 1]
```diff
@@ -26,11 +31,20 @@
      */
     @PostMapping("/compilePython")
```
Comment (JorgeRCV) [inline src/main/java/org/fundacion/jala/converter/controller/PythonCompilerController.java:32]: Java doc is missing in this class

File: src/main/java/org/fundacion/jala/converter/controller/PythonCompilerController.java [hunk 2]
```diff
@@ -26,11 +31,20 @@
      */
     @PostMapping("/compilePython")
     public String uploadFile(@RequestParam("file") MultipartFile file, @RequestParam("version") String version) throws IllegalStateException, IOException {
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/controller/PythonCompilerController.java]: the parameters should be final

File: src/main/java/org/fundacion/jala/converter/controller/PythonCompilerController.java [hunk 3]
```diff
@@ -26,11 +31,20 @@
...
+        PythonCompiler pythonCompiler = new PythonCompiler();
+        LOGGER.info("finish");
+        return pythonCompiler.compiler(Python.V3, storagePath);
+    }
+
+    /**
+     * Endpoint for proving compile python
+     */
+    @GetMapping("/compilePython2")
+    public String uploadFile2(){
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/controller/PythonCompilerController.java]: the parenthesis should have a space before

**Review by Raymundogs — APPROVED** [review]

**Review by jpgonzales27 — APPROVED** [review]

**Review by JorgeRCV — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Comment by cristianfchq** [comment]
Good Job

**Pull request merged**